docs/configuration.rst

```diff
@@ -58,6 +58,11 @@ ACCOUNT_LOGOUT_ON_GET (=False)
   mere GET request. See documentation for the `LogoutView` for
   details.
 
+ACCOUNT_LOGOUT_ON_PASSWORD_CHANGE (=False)
+  Determines whether or not the user is automatically logged out after
+  changing the password. See documentation for `Django's session invalidation on password change` 
```

Comment:
Nit: mind the line wrapping. It'd also help to link to [Django docs on the topic](https://docs.djangoproject.com/en/1.7/topics/auth/default/#session-invalidation-on-password-change).